### PR TITLE
Simple search unit test

### PR DESCRIPTION
**Pull request by Lumiere-Lodi**
This PR add unit test for simple search children components.

### CODE REVIEW

File: src/test/SearchDateInput.test.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { render, screen } from '@testing-library/react'
+import SearchDateInput from '../components/search/SearchDateInput'
+
+test('Show correct label when rendered', () => {
+  const { container } = render(
```
Comment (MatthewErispe) [inline src/test/SearchDateInput.test.tsx:5]: @Lumiere-Lodi should this not be wrapped in act? Getting warning errors

**Review by MatthewErispe — CHANGES_REQUESTED** [review]

File: src/test/SearchDateInput.test.tsx (hunk 1 above)
Comment (Lumiere-Lodi) [reply src/test/SearchDateInput.test.tsx:5]: it was happening in the search file. FIXED now. You can test again.

**Review by MatthewErispe — APPROVED** [review]
@Lumiere-Lodi I am happy with the additional changes made

**Review by MatthewErispe — APPROVED** [review]
@Lumiere-Lodi LGTM

**Review by IlyesBenAmara — APPROVED** [review]
LGTM :technologist:

**Pull request merged**